paperqa/agents/main.py

```diff
@@ -140,6 +143,11 @@ async def run_fake_agent(
     ) = None,
     **env_kwargs,
 ) -> tuple[Answer, AgentStatus]:
+    if query.settings.agent.max_timesteps is not None:
```

Comment:
jc -- why are we calling the agent steps timesteps? I think of timesteps from physical models (i.e. molecular dynamics) where each iteration is a unit of time, like 4 picoseconds or something. This is more like actionsteps in my mind.

Reply:
I hear what you're saying.

I think one can also encounter vagaries with "step", for example one can wonder does "step" mean:
- One step = agent selection + environment step
- Two steps = agent selection + environment step

I went with timestep to exactly match `ldp.data_structures.Transition.timestep`

Reply:
Let me know if you think we should name it otherwise